app/src/main/res/values/strings.xml

```diff
@@ -110,6 +109,10 @@
     <string name="please_specify">Please specify!</string>
     <string name="please_select_at_least_one">Please select at least one!</string>
     <string name="you_can_t_select_more_than_3">You can not select more than 3!</string>
+    <string name="set_password">Enter Password</string>
```

Comment:
Some added strings seem to not be used anywhere (Cirrus warnings).

Reply:
Right I'll remove it in my next branch (I don't have the energy to wait another 25 minutes for cirrus to build)